Dockerfile

```diff
@@ -3,7 +3,7 @@ ARG USER_ID=3001
 ARG USER_NAME=sage
 ARG SOURCE_DIR=/$USER_NAME/source
 ARG NPM_REGISTRY=upstream
-ARG NODE_ENV=development
```

Comment:
I would get this confirmed with Micah. So far, this has been the default and its also environment specific. The modification is prominent during the deployment phase ( and thats environment specific)

Reply:
I spoke with Micha and made the corrected changes.